Submitted By: DJ Lucas <dj AT linuxfromscratch DOT org>
Date: 2004-11-27
Initial Package Version: 1.4.1
Origin: Sent by Sourcemage Guru Hamish Greig
Description: Motif compilation fix. Needed if default optimization flags 
             are used.

$LastChangedBy$
$Date$


--- j2sdk-build-orig/motif/lib/Xm/util/mkmsgcat.c	2004-10-19 13:44:16.000000000 -0500
+++ j2sdk-build/motif/lib/Xm/util/mkmsgcat.c	2004-11-15 20:22:21.000000000 -0600
@@ -14,7 +14,7 @@
 #include <string.h>
 char buf[1024];
 
-char *OSF_COPYRIGHT = "$\n\
+char *OSF_COPYRIGHT = "\
 $  @OSF_COPYRIGHT@\n\
 $  COPYRIGHT NOTICE\n\
 $  Copyright (c) 1990, 1991, 1992, 1993 Open Software Foundation, Inc.\n\
@@ -28,7 +28,7 @@
 $ (c) Copyright 1996 Novell, Inc. \n\
 $ (c) Copyright 1996 FUJITSU LIMITED.\n\
 $ (c) Copyright 1996 Hitachi.\n\
-$";
+";
 
 char *OSF_COPYRIGHT_C = "/*\n\
  *\n\
